migration-engine/core/src/core_error.rs

```diff
@@ -2,13 +2,13 @@ use migration_connector::{ConnectorError, ListMigrationsError};
 use std::{error::Error as StdError, fmt::Display};
 use user_facing_errors::{KnownError, UserFacingError};
 
-/// The result type for migration engine commands.
+/// The result type for migration engine commands
```

Comment:
?

Reply:
For consistency (single-sentence comments don't have a period in most places). Do you think it reads better with the period?

Reply:
Just wondering. Should we have a 30-60 minute long chat about this in the next engines meeting :D I'm actually fine with both, but grammatically it would be correct to have punctuation.

Reply:
We definitely should! I love bikeshedding